kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -368,6 +368,15 @@ private void translateSorts(Set<SortHead> tokenSorts, Map<String, Boolean> attri
 
     private void translateSymbols(Map<String, Boolean> attributes, SetMultimap<KLabel, Rule> functionRules,
                                   Set<KLabel> impurities, Set<Production> overloads, StringBuilder sb) {
+        HashMap<String, String> KLabelToKORE = new HashMap<String, String>();
```

Comment:
I don't understand why we're using this instead of just calling `production` on the klabel to get the production corresponding to the klabel and then calling convert directly in the function for converting attributes. It seems simpler to me than constructing this map.

Reply:
@dwightguth 
From what I can see, `production` takes as argument a `KApply`. All we have in this scenario is a string denoting the KLabel. I couldn't figure out a way to extract a `Production` from that alone...

Reply:
So, you can do something like `KApply(KLabel(str))`